gobblin-runtime/src/main/java/gobblin/runtime/CommandLineLauncher.java

```diff
@@ -0,0 +1,49 @@
+package gobblin.runtime;
```

Comment:
Actually, can you remove this class?
A better version will be added in a different PR (with the partitioned writer).